### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExplorer_Keyboard: Tooltip is not visible to keyboard only users but only visible on mouse-hover.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51995]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.1.1, MAS 2.1.1
Tools Used : Keyboard

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Explorer"( Cntrl + Shift + E ) button.
3. Navigate to "Horizontal Group Layout" , "Save All" , "Collapse All Editors", " New File", " New Folder", "Refresh", " Collapse All" buttons using "Tab".
4. Also, mouse hover on all above mentioned buttons.

**Actual:**
Tooltip information is not available for keyboard only users for "Horizontal Group Layout" , "Save All" , "Collapse All Editors", "New File", "New Folder", "Refresh", "Collapse All" buttons which is available on mouse hover.

**Expected:**
Tooltip information should be made available for keyboard only users which is available using mouse.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard only users will not be able to gain information regarding few controls present on screen which is only avaiable using mouse hover.

**MAS Reference:**
MAS 2.1.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

MAS 1.1.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Attachment for Reference:** 
[A11y_VSCode_ViewExplorer_Keyboard_TooltipIssue.pptx](https://github.com/Microsoft/vscode/files/2105487/A11y_VSCode_ViewExplorer_Keyboard_TooltipIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52016
